### PR TITLE
Paul/mtpelerin pt2

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [x] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206626267967662

### CODE REVIEW

**Review by samholmes — CHANGES_REQUESTED** [review]

File: src/plugins/gui/providers/mtpelerinProvider.ts [hunk 1]
```diff
@@ -11,25 +17,27 @@ import {
...
   widget: {
-    prod: '',
-    test: ''
+    prod: 'https://widget.mtpelerin.com',
+    test: 'https://widget-staging.mtpelerin.com'
   }
 }
 
-const MODE = 'prod'
+const MODE = Math.random() < 10 ? 'prod' : 'test'
```
[the remark this answers is not in the document]
Comment (paullinator) [reply src/plugins/gui/providers/mtpelerinProvider.ts:40]: No. If I hard code one of the values, the `if` checks cause a TS error.

File: src/plugins/gui/providers/mtpelerinProvider.ts [hunk 2]
```diff
@@ -40,10 +48,34 @@ const PLUGIN_TO_CHAIN_ID_MAP: { [pluginId: string]: string } = {
...
+  polygon: true,
+  rsk: true,
+  zksync: true
+}
+
+if (MODE === 'test') {
+  delete PLUGIN_TO_CHAIN_ID_MAP.ethereum
+  PLUGIN_TO_CHAIN_ID_MAP.goerli = 'mainnet'
+  PLUGIN_EVM_MAP.goerli = true
+}
```
[the remark this answers is not in the document]
Comment (paullinator) [reply src/plugins/gui/providers/mtpelerinProvider.ts:77]: I'll add a new commit to do so.

File: src/plugins/gui/providers/mtpelerinProvider.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (samholmes) [reply src/plugins/gui/providers/mtpelerinProvider.ts]: `const MODE: 'prod' | 'test' = 'prod'` didn't work?

**Review by samholmes — APPROVED** [review]

File: src/plugins/gui/providers/mtpelerinProvider.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (paullinator) [reply src/plugins/gui/providers/mtpelerinProvider.ts]: Nope. didn't work

**Pull request merged**